fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/traci/reader/TrafficLightProgramTraciReader.java

```diff
@@ -61,6 +61,7 @@ protected SumoTrafficLightLogic readFromStream(DataInputStream in) throws IOExce
         for (int i = 0; i < numberOfParams; i++) {
             paramReader.readFromStream(in); // key/value pair as list with two items
         }
+        numBytesRead += paramReader.getNumberOfBytesRead();
```

Comment:
does this work for multiple parameters? I think `numberOfBytesRead` returns only the the number of bytes read from the most previous call of `readFromStream`.

Reply:
Seems to also work for multiple parameters, I added a extra test parameter

Reply:
So the way this works is, that each param pair (key, value) is read using the `ListTraciReader`'s `readFromStream(...)` method.
This means it is correct, that we call it in a for-loop.